demos/src/content/3-party-table.mustache

```diff
@@ -13,7 +13,7 @@
 </ol>
 
 <h2 id="this-is-a-table">This is a table</h2>
-<table class="o-table o-table o-table--row-stripes" data-o-component="o-table">
+<table class="o-table o-table o-table--row-stripes o-layout__main--single-span" data-o-component="o-table">
```

Comment:
BEM whoops `o-layout__main__single-span`

Reply:
that would make `o-layout__main--full-span` wrong also.

Reply:
Good spot, could we update both but keep `o-layout__main--full-span` also for backwards compatibilty?